framework/src/modules/chain/submodules/blocks/process.js

```diff
@@ -347,28 +347,11 @@ class Process {
 	/**
 	 * Handle newly received block.
 	 *
-	 * @listens module:transport~event:receiveBlock
 	 * @param {block} block - New block
 	 * @todo Add @returns tag
 	 */
 	// eslint-disable-next-line class-methods-use-this
-	onReceiveBlock(block) {
```

Comment:
Are we going to remove this `on<Event>` convention? If that's the case then there are many `onBind` handlers in the code.

Reply:
yes we are going to remove `bus` in general, but step by step